docs/intro/tutorial02.txt

```diff
@@ -3,602 +3,472 @@ Writing your first Django app, part 2
 =====================================
 
 This tutorial begins where :doc:`Tutorial 1 </intro/tutorial01>` left off. We're
-continuing the Web-poll application and will focus on Django's
-automatically-generated admin site.
+continuing the Web-poll application and will focus on creating the public
+interface -- "views."
 
-.. admonition:: Philosophy
-
-    Generating admin sites for your staff or clients to add, change and delete
-    content is tedious work that doesn't require much creativity. For that
-    reason, Django entirely automates creation of admin interfaces for models.
-
-    Django was written in a newsroom environment, with a very clear separation
-    between "content publishers" and the "public" site. Site managers use the
-    system to add news stories, events, sports scores, etc., and that content is
-    displayed on the public site. Django solves the problem of creating a
-    unified interface for site administrators to edit content.
-
-    The admin isn't intended to be used by site visitors. It's for site
-    managers.
-
-Creating an admin user
-======================
-
-First we'll need to create a user who can login to the admin site. Run the
-following command:
-
-.. code-block:: console
-
-    $ python manage.py createsuperuser
-
-Enter your desired username and press enter.
+Philosophy
```

Comment:
Didn't see this was copied from the existing tutorial, but yes, I think "Overview" is a better word here. The other instances of `.. admonition:: Philosophy` seem okay as is.